cross/java-11-openjdk/Makefile

```diff
@@ -75,6 +76,8 @@ CONFIGURE_ARGS += --disable-hotspot-gtest
 # Disable all GUI related
 CONFIGURE_ARGS += --enable-headless-only
 
+COMPILE_MAKE_OPTIONS += JOBS=$(NCPUS)
```

Comment:
It automagically detects how many jobs should run, so you can remove the line

Reply:
aaahhh, that I didn't know... still, i'd add it by default to be consistent but its just me.
@publicarray  and anyhow, I guess I can cancel this PR as yours superseeds it?